### PR TITLE
Add Agent Conference to conferences docs

### PR DESCRIPTION
**Pull request by brentvatne**
It will be fun, we will go skiing and talk about React, React Native, Jest, Elm, and fun things.

### CODE REVIEW

File: docs/community/conferences.md [hunk 1]
```diff
@@ -83,3 +83,8 @@ October 26-28 in Bratislava, Slovakia
 October 26-28 online
 
 [Website](https://allremoteconfs.com/react-2016) - [Schedule](https://allremoteconfs.com/react-2016#schedule)
+
+### Agent Conference 2017
+January 20-21 in Dornbirn, Austria
+
+[Website](http://agent.sh/)
```
Comment (Aweary) [inline docs/community/conferences.md]: Can you put this under the "Upcoming Conferences" section at the top of the file? Thanks! 😄

**Comment by brentvatne** [comment]
Sorry @Aweary, that was silly of me, thanks for catching it

**Comment by Aweary** [comment]
No worries, easy mistake to make 😄

**Comment by gaearon** [comment]
@Aweary Please don't forget to put "needs merge to stable" label 😉 . I would have missed this change if it didn't conflict with another change later.

**Comment by Aweary** [comment]
@gaearon I was looking for this PR! I knew I missed one, but couldn't find which. Thanks.

**Comment by Aweary** [comment]
@gaearon I wonder if we can use http://danger.systems/ to either add the label or comment with a reminder to add the label whenever docs have been changed?

**Pull request merged**